index.js

```diff
@@ -0,0 +1,11 @@
+const button = document.querySelector('.menu-bars');
+const mobileNavCloseButton = document.querySelector('.nav-close');
+
+const mobilenav = document.getElementById('mobile-nav');
+button.onclick = function () {
+  mobilenav.style.display = 'block';
+};
+
+mobileNavCloseButton.onclick = function () {
+  mobilenav.style.display = 'none';
+};
```

Comment:
Please consider adding the same functionality to the Nav Items so that when the user clicks any of the Nav Items it not only reaches the correct destination but also closes the menu. For better demonstration, I am attaching the demonstration for this.


https://user-images.githubusercontent.com/94223367/211090747-492429f4-4b66-4147-9155-b8bf2ededdf7.mp4